### DISCUSSION TITLE
Dual-licensing verification

### DISCUSSION THREAD

**Discussion by lambdakilo** [cesanta/mongoose#2318] [Ask questions, get answers] [answered]
Could we have some sort of verification computed automatically that the contributors before 2013 have signed the CLA so that the GPL would not be violated? From my understanding, you can commerical-license GPL code only if the original contributors give permission i.e. sign the CLA: https://softwareengineering.stackexchange.com/questions/331846/dual-license-with-gpl-and-a-closed-source-license

Here's when the dual-licensing happened: https://github.com/cesanta/mongoose/commit/218428662e6f8d30a83cf8a89f531553f1156d25

Thanks ❤️

**Comment by scaprile** [comment, marked as answer]
Please contact Cesanta for that
https://cesanta.com/contact.html
